src/docs/release/breaking-changes/index.md

```diff
@@ -14,14 +14,16 @@ release, and listed in alphabetical order:
 
 * [Android v1 embedding app and plugin creation deprecation][]
 * [Material Chip button semantics][]
-* [Network Policy on iOS and Android][]
+<!-- * [Network Policy on iOS and Android][] -->
 * [The new Form, FormField auto-validation API][]
+* [Android FlutterMain.setIsRunningInRobolectricTest testing API removed][]
 
 [Android v1 embedding app and plugin creation deprecation]: /docs/release/breaking-changes/android-v1-embedding-create-deprecation
 [Material Chip button semantics]: /docs/release/breaking-changes/material-chip-button-semantics
 [The new Form, FormField auto-validation API]: /docs/release/breaking-changes/form-field-autovalidation-api
 <!-- Re-enable once HTTP ban lands to flutter/flutter -->
 <!-- [Network Policy on iOS and Android]: /docs/release/breaking-changes/network-policy-ios-android -->
+[Android FlutterMain.setIsRunningInRobolectricTest testing API removed]: /docs/release/breaking-change/android-setIsRunningInRobolectricTest-removed
```

Comment:
This should be `breaking-changes` instead of `breaking-change` I think, which is why the linkcheck tool is failing.

Reply:
Doh! 🤦🏻‍♂️
Thanks John